packages/fluent-theme/src/components/typingIndicator/SlidingDotsTypingIndicator.tsx

```diff
@@ -0,0 +1,24 @@
+import { testIds } from 'botframework-webchat-component';
+import { useStyles } from 'botframework-webchat-styles/react';
+import cx from 'classnames';
+import React, { memo } from 'react';
+
+import { useVariantClassName } from '../../styles';
+import SlidingDots from '../assets/SlidingDots';
+import styles from './SlidingDotsTypingIndicator.module.css';
+
+function SlidingDotsTypingIndicator() {
+  const classNames = useStyles(styles);
+  const variantClassName = useVariantClassName(classNames);
```

Comment:
nit: not needed, no variant classnames in the `classNames` object available

Reply:
Let me fix it in another PR.